### ISSUE TITLE
[java] False positive on DontImportJavaLang when importing Java 7's java.lang.invoke.MethodHandles

### ISSUE THREAD

**Issue by jsotuyod** [pmd/pmd#339]
**Rule Set:** java-imports/DontImportJavaLang

**Description:**

**Code Sample demonstrating the issue:**

```
import java.lang.invoke.MethodHandles;

public class Foo {
  private MethodHandles mh;
}
```

**Running PMD through:** *Any*
